gravityflow/gravityflow
# Fix HTML Field Conditional Logic on User Input Workflow Step

## Description
Re: [HS#18933](https://secure.helpscout.net/conversation/1694288950/18933?folderId=3509658)
For HTML fields the field conditional logic is not honored by the User Input step unless the step is submitted.

## Testing instructions
1. Create a form with text field A, and HTML fields B and C. Add content to HTML fields.
2. Update HTML field B with conditional logic to display only if text field A is empty.
3. Update HTML field C with conditional logic to display only if text field A is not empty.
4. Create a Workflow User Input Step with text field A as editable.
5. Create an entry for the form.
6. Go to the User Input step for the entry, confirm the conditional logic does not conform for the HTML fields B & C if we are editing content on the text field A.
7. Switch to this branch, and repeat step 6. Confirm the conditional logic does work as expected now.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
